lib/active_merchant/billing/gateways/paybox_direct.rb

```diff
@@ -135,8 +148,6 @@ def parse(body)
       end
 
       def commit(action, money = nil, parameters = nil)
-        parameters[:montant] = ('0000000000' + (money ? amount(money) : ''))[-10..-1]
-        parameters[:devise] = CURRENCY_CODES[options[:currency] || currency(money)]
```

Comment:
Wondering why this isn't just blowing up on `undefined local variable or method`